.github/workflows/test_conda_build.yaml

```diff
@@ -54,5 +54,4 @@ jobs:
           conda build nebari
       - name: Test nebari
         run: |
-          conda install --use-local nebari
```

Comment:
@dcmcand why did you remove the install? Isn't it needed for `nebari` to work?

Reply:
It was installing the wrong version.